packages/tizen_rpc_port/lib/bundle.dart

```diff
@@ -0,0 +1,272 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:collection';
+import 'dart:ffi';
+import 'dart:typed_data';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/services.dart';
+import 'package:tizen_interop/4.0/tizen.dart';
+
+/// Bundle is a string based Dictionary ADT.
+/// A dictionary is an ordered or unordered list of key element pairs,
+/// where keys are used to locate elements in the list.
+class Bundle extends MapMixin<String, Object> {
```

Comment:
This file will be removed after https://github.com/flutter-tizen/plugins/pull/428 is applied.